### ISSUE TITLE
CPAP octopus pro 

### ISSUE THREAD

**Issue by bsilly1** [clowrey/x5sa_pro_klipper#1]
where did you connect power to? I went off your GitHub page. But when I power on it goes full blast until klipper turns on? I went to the power supply currently.

**Comment by clowrey** [comment]
Yes, that is because of how the GPIO pin defaults are set in Klipper firmware - I believe we can fix it by compiling the firmware with a pin over ride turned on for whatever pin you use on the fan. - I haven't tried yet -

When you do the "make menuconfig"  we need a custom setting of "GPIO pins to set at startup" 

If you experiment and get it to work definitely let me know!
